doc/source/whatsnew/v0.22.0.txt

```diff
@@ -262,4 +262,4 @@ Other
 - Fixed a bug where creating a Series from an array that contains both tz-naive and tz-aware values will result in a Series whose dtype is tz-aware instead of object (:issue:`16406`)
 - Fixed construction of a :class:`Series` from a ``dict`` containing ``NaN`` as key (:issue:`18480`)
 - Adding a ``Period`` object to a ``datetime`` or ``Timestamp`` object will now correctly raise a ``TypeError`` (:issue:`17983`)
--
```

Comment:
move to conversion (prob should move some of the other ones appropriately as well) but other PR for that

Reply:
Yep, done.